torch/jit/frontend.py

```diff
@@ -252,12 +252,23 @@ def build_Print(ctx, stmt):
 
 
 class ExprBuilder(Builder):
-    binop_map = {
-        ast.Add: '+',
-        ast.Sub: '-',
-        ast.Mult: '*',
-        ast.Div: '/',
-    }
+    if PY2:
```

Comment:
Let's not duplicate the entries in this map that are the same across both pythons. This will lead to bugs in the future.

```
binop_map = ...
if not PY2:
   binop_map[ast.MatMult] = '@'
```